src/RestEase/RestEase.csproj

```diff
@@ -33,19 +33,22 @@
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.1' ">
     <PackageReference Include="System.Net.Http" Version="4.1.0" />
     <PackageReference Include="System.Collections.Concurrent" Version="4.0.12" />
+    <PackageReference Include="System.ComponentModel.Annotations" Version="4.3.0" />
```

Comment:
I hadn't clocked `DisplayAttribute` was in a different assembly. I'm not sure I'm comfortable adding another package reference for something that 99.9% of people aren't going to be using.

Reply:
Right, so here's what attributes we have available in each env:

| Framework | `EnumMember` | `DisplayName` | `Display` |
| --------- | ------------ | ------------- | --------- |
| `net45`   | ✔️           | ✔️           | ❌ (System reference) |
| `netstandard1.1` | ❌ | ❌ | ❌ (NuGet package) |
| `netstandard2.0` | ✔️ | ✔️ | ❌ (NuGet package) |

So without a NuGet package, .NET Standard 1.1 code will not have a way to specify a display name with the default serializer (they can add their own custom one to do whatever they want of course).

If that is fine, we could remove the NuGet packages? (The reference for `net45` should be fine since that is available in the framework and won't require pulling anything extra.)